docs/strictdoc_01_user_guide.sdoc

```diff
@@ -269,12 +301,71 @@ optional configuration fields and an optional ``[FREETEXT]`` block.
 [/FREETEXT]
 
 [SECTION]
+UID: DOCUMENT_FIELD_OPTIONS
+TITLE: Document configuration options
+
+[FREETEXT]
+The ``OPTIONS`` field may have the following attribute fields:
+
+.. list-table:: SDoc grammar ``DOCUMENT``-``OPTIONS`` fields
+   :widths: 20 80
+   :header-rows: 1
+
+   * - **Field**
+     - **Attribute values**
+
+   * - ``MARKUP``
+     - ``RST``, ``HTML``, ``Text``
+
+   * - ``AUTO_LEVELS``
+     - ``On``, ``Off``
+
+   * - ``REQUIREMENT_STYLE``
+     - ``Inline``, ``Table``
+
+   * - ``REQUIREMENT_IN_TOC``
+     - ``True``, ``False``
+
+[/FREETEXT]
+
+[SECTION]
+LEVEL: None
```

Comment:
I understand the intention, but I would prefer all sections of this document to be included with numbers into TOC. 

My context: It is easier for me to avoid reasoning about similar cases when maintaining this user guide. If all sections have levels and numbers, there is nothing to think about. Let's keep this feature for user space documents when a user is forced to have the `None` levels.

Reply:
>  Maybe it is good idea to remove the "OPTION: " part and use the real Uppercase attribute/field name as title. I.e. the same usage as for all other fields.

Sorry, I don't understand what you are referring to by `remove the "OPTION:" part"` - which option is this and where?

Reply:
Just the text "Option:" in the titles e.g. "Option: Requirement style" => "REQUIREMENT_STYLE" etc.

Reply:
Ah! Now I see. Sure, go ahead.